### ISSUE TITLE
Hook BatchNorm 

### ISSUE THREAD

**Issue by kamathhrishi** [OpenMined/PySyft#2175]
Following issue #2132 we should hook batchnorm. Since in general batchnorm is very common in training neural networks.
